backend/tracim_backend/lib/mail_notifier/utils.py

```diff
@@ -1,11 +1,12 @@
 class SmtpConfiguration(object):
     """Container class for SMTP configuration used in Tracim."""
 
-    def __init__(self, server: str, port: int, login: str, password: str):
+    def __init__(self, server: str, port: int, login: str, password: str, use_implicit_ssl: bool):
```

Comment:
I suggest a default value to False + a docsting explaining what means use_implicit_ssl

Reply:
not required at the moment, actually: a refactor will arrive soon.